components/ledger/internal/bootstrap/config.rabbitmq.go

```
@@ -118,6 +126,15 @@ func initMultiTenantRabbitMQ(
 		return nil, fmt.Errorf("TenantClient is required for multi-tenant RabbitMQ initialization")
 	}
 
+	if redisConnection == nil {
+		return nil, fmt.Errorf("Redis connection is required for multi-tenant RabbitMQ consumer (Redis Pub/Sub)")
```

golangci-lint (2.11.3)

[error] ST1005: error strings should not be capitalized

(staticcheck)